var/spack/repos/builtin/packages/netlib-blas/package.py

```diff
@@ -0,0 +1,48 @@
+from spack import *
+import os
+
+
+class NetlibBlas(Package):
+    """Netlib reference BLAS"""
+    homepage = "http://www.netlib.org/lapack/"
+    url      = "http://www.netlib.org/lapack/lapack-3.5.0.tgz"
+
+    version('3.5.0', 'b1d3e3e425b2e44a06760ff173104bdf')
+
+    variant('fpic', default=False, description="Build with -fpic compiler option")
+    variant('fortran', default=False, description="Build with Fortran support")
```

Comment:
@tgamblin what do you think in terms of consistency to other blas/lapack packages (`openblas`, `atlas`, `netliblapack`)?